src/vaishnaviVJun20_arrayExamples/Fibonacci.java

```diff
@@ -0,0 +1,24 @@
+package vaishnaviVJun20_arrayExamples;
+
+//To print first n Fibonacci Numbers
+
+public class Fibonacci {
+	
+	void printFibonacciNumbers(int input) {
+		int sum=0,num1=0,num2=1;
+		System.out.println("Input number is:" +input);
+		System.out.print(num2);
```

Comment:
If printing sum no need to print num2.

Reply:
-- Printing num2 for required 11 numbers output.
--Let me know your thoughts on this